src/modelinstance.cpp

```diff
@@ -685,6 +686,12 @@ Status ModelInstance::loadOVCompiledModel(const ModelConfig& config) {
             config.getTargetDevice());
         return status;
     }
+    auto status = validatePluginConfiguration(config.getPluginConfig(), config.getTargetDevice(), ieCore);
```

Comment:
Why do we validate configuration twice? Once after config parse and here after model compilation?